README.rst

```diff
@@ -16,7 +16,7 @@ Modin can be installed with pip: ``pip install modin``
 Pandas on Ray
 -------------
 
-*Pandas on Ray is a project designed to effortlessly scale your pandas code requiring a change of only a single line of code*
+*Pandas on Ray is a project designed to effortlessly scale your pandas code by changing only a single line of code*
```

Comment:
I played with this sentence too much. The simplest is probably the best:

`*Pandas on Ray is a library that allows you to effortlessly scale pandas by changing only a single line of code*`